js/main.js

```diff
@@ -26,6 +18,75 @@ function getRandomArbitrary(min, max, digits) {
   }
   return('Задан неверный диапазон! Укажите другие числа.');
 }
-// числа к примеру
-getRandomArbitrary(1, 1, 5);
-// Источник https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+
+const type = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+const checkin = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+const checkout = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+const features = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner',
+];
+const photos = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const getRandomItems = (items, count) => items.slice(0, count);
+
+const createOffer = () => {
+  const avatarNumber = randomIntegerAuthor(1, 9);
+  const locationLat = getRandomArbitrary(15, 70, 5);
+  const locationLng = getRandomArbitrary(100, 180, 5);
+  const typeElement = randomIntegerAuthor(1, features.length - 1);
+  const checkinElement = randomIntegerAuthor(1, checkin.length - 1);
+  const checkoutElement = randomIntegerAuthor(1, checkout.length - 1);
+
+  return {
+    author: {
+      avatar: `img/avatars/user0${avatarNumber}.png`,
+    },
+
+    offer: {
+      title: 'Offer',
+      address: `${locationLat}, ${locationLng}`,
+      price: randomIntegerAuthor(400, 5000),
+      type: type[typeElement],
+      rooms: randomIntegerAuthor(1, 3),
+      guests: randomIntegerAuthor(1, 5),
+      checkin: checkin[checkinElement],
+      checkout: checkout[checkoutElement],
+      features: getRandomItems(features, randomIntegerAuthor(0, features.length - 1)),
+      description: 'Очень комфортные комнаты по выгодным ценам',
+      photos: getRandomItems(photos, randomIntegerAuthor(0, photos.length - 1)),
+    },
+
+    location: {
+      lat: locationLat,
+      lng: locationLng,
+    },
+  };
+};
+
+const offers = new Array(10).fill(null).map(() => createOffer());
+
+console.log(offers);
```

Comment:
Я не смогу принять, если линтер будет ругать, по этому давай поступим так. Сделаем ещё функцию, которая будет генерить обьявления, в зависимости от переданного параметра:

```suggestion
const createOffers = (count) => {
  const offers = new Array(count).fill(null).map(() => createOffer());
  
  return offers;
};

createOffers(10)
```
Надеюсь ясно что тут происходит 🙏